### ISSUE TITLE
Flutter-beta build failure: 2.11 (is next release version) 

### ISSUE THREAD

**Issue by Dall127** [bitsdojo/bitsdojo_window#148]
When using 'Flutter 2.11.0-0.1.pre', the build fails at 

```
: Error: 'Size' is imported from both 'dart:ffi' and 'dart:ui'.
    Size sizeToSet = Size(width, height);
                     ^^^^
```
at  ../…/src/window.dart:228

This isn't the only line failing in this file, but it seems to be limited to this file, as building for the web on this branch of beta works.

**Comment by A1000carZ** [comment]
Same issue.

**Comment by DuttaAB-dev** [comment]
Same issue with flutter master 2.11.0-0.0.pre.537

**Comment by bostrot** [comment]
Same issue. Reverting back to v2.8 seems to work:

    flutter channel flutter-2.8-candidate.20
    flutter upgrade

**Comment by simonbengtsson** [comment]
PR fixing this issue: https://github.com/bitsdojo/bitsdojo_window/pull/149. Can be used like this:

```
  # https://github.com/bitsdojo/bitsdojo_window/issues/148
  bitsdojo_window:
    git:
      url: https://github.com/simonbengtsson/bitsdojo_window
      path: bitsdojo_window
      ref: master
```

**Comment by bitsdojo** [comment]
Fixed in `bitsdojo_window: ^0.1.2` - https://github.com/bitsdojo/bitsdojo_window/commit/f320b623db107b03a5976fd88e52ec4fe81fa0c9

Closing this issue. Please re-open if needed.
